backend/handler/passcode.go

```diff
@@ -453,7 +452,7 @@ func (h *PasscodeHandler) Finish(c echo.Context) error {
 
 func (h *PasscodeHandler) GetSessionToken(c echo.Context) jwt.Token {
 	var token jwt.Token
-	sessionCookie, _ := c.Cookie("hanko")
+	sessionCookie, _ := c.Cookie(h.cfg.Session.Cookie.GetName())
```

Comment:
@FreddyDevelop how can we handle the errors here? all errors in GetSessionToken are ignored. They should be logged at least. What is the best way to get a logger instance here?

That would have helped to faster find that bug, because the accessed cookie doesn't exist.

Reply:
We do not log the error, that a cookie is missing, because the session token can also be send within the `Authorization` header and when the error gets logged then we would clutter the logs with unnecessary messages. 

Only solution to that would be to log that errors only when no token can be returned. You can use zerolog for the logging. An example can be found at: https://github.com/teamhanko/hanko/blob/579cef31a136c6cf3cb650d6e885f9595c8cda2c/backend/handler/passcode.go#L234